sdk/typespec/typespec_client_core/src/macros.rs

```diff
@@ -189,27 +290,63 @@ mod test {
     }
 
     #[test]
-    fn test_color_parse_1() {
+    fn color_parse_1() {
```

Comment:
Note: I don't like "test_" prefixes when in a "test(s)" module. It's redundant. "test_" is great when tests are in the same module e.g.,

```rust
fn foo() {}

#[test]
fn test_foo() {}
```